wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -6,43 +6,46 @@
     <div class="ds-card" id="election_{{ postelection.election.slug }}">
         <div class="ds-card-body">
 
-            {% if postelection.cancelled %}
```

Comment:
Unfortunately I think the change to remove this and move it further down to line 96 has had the knock on effect of breaking some historic elections e.g. those cancelled due to covid in 2020. See the difference between the [live site](https://whocanivotefor.co.uk/elections/gla.a.2020-05-07/london-assembly-elections-additional/) and [dev site](https://dev.wcivf.club/elections/gla.a.2020-05-07/london-assembly-elections-additional/)

I think the reason this change was made was so that the candidates were still listed when a ballot was uncontested? So I wonder if this should be reverted, and then instead we take

Reply:
I made this change as suggested, but then lost the date and summary line at the top: "X candidates stood in the ..." since it's now included in the `else` (not cancelled). I'll look again tomorrow with fresh eyes to avoid further duplication. 👀